sstables/sstables.hh

```diff
@@ -123,6 +123,14 @@ constexpr const char* snapshots_dir = "snapshots";
 constexpr const char* quarantine_dir = "quarantine";
 constexpr const char* pending_delete_dir = "pending_delete";
 
+constexpr std::array<const char*, 5> table_subdirectories = {
```

Comment:
If the "5" here bothers you (like it did me when I saw it ;-)), you can also use std::to_array - see https://en.cppreference.com/w/cpp/container/array/to_array. It's new in C++20.

Reply:
Plain std::array without the template parameters will work just as well.

Reply:
But, better to move it out of the header.